styles.scss

```diff
@@ -554,21 +562,28 @@ footer {
     width: 22rem;
     height: auto;
   }
-  .w-50 {
-    width: 99% !important;
+  .w-50.w-50 {
```

Comment:
bootstrap classes aren't supposed to be modified like that, if you want a custom width try to implement it with a custom class or target in a different way your element